src/frontend/editor.tsx

```diff
@@ -37,19 +34,14 @@ export interface EditorProps {
 }
 
 export function Editor({ state, providerUrl, ltik }: EditorProps) {
-  // HACK: Change strings in link element. Searching or inserting an id is not possible in this integration.
-  loggedInDataDe.strings.editor.plugins.text.linkOverlay.placeholder =
-    'https://example.com/'
-  loggedInDataDe.strings.editor.plugins.text.linkOverlay.inputLabel =
-    "Gib eine URL inklusive 'https://' ein"
-
   return (
-    <SerloEditorPackage
-      initialState={state.document}
-      instanceData={instanceDataDe as SerloEditorProps['instanceData']}
-      loggedInData={loggedInDataDe as SerloEditorProps['loggedInData']}
-    >
-      {(editor) => {
+    <SerloEditorPackage initialState={state.document}>
+      {(editor, languageData) => {
+        // HACK: Change strings in link element. Searching or inserting an id is not possible in this integration.
```

Comment:
Not even sure if I would consider this a hack. I mean we need some way of overwriting default strings, but also don't encourage it…

Reply:
Agreed, just thought that Lars had some reason to mark it as a hack, so I didn't want to remove the comment without consulting him.

Reply:
I think the readability would improve if we got rid of the 'hack'. I had a quick look into the code and in `link-controls.tsx` we have boolean `isSerloLinkSearchActive` which could control what strings are displayed internally. I'll make the change if there is no veto.